CMakeLists.txt

```diff
@@ -108,6 +108,19 @@ if(HAIKU)
   set(UBSAN OFF)
 endif()
 
+if(AMIGA)
+  set(ASAN OFF)
+  set(UBSAN OFF)
+  set(NONET ON)
+  set(USE_SDL1 ON)
+  find_package(Freetype REQUIRED)

```

Comment:
Why do we need to find these 2 only for Amiga?

Reply:
Libs are statically linked on Amiga.

Reply:
Perhaps we should add a more general `STATIC` option?

Reply:
Libs are also linked statically on the Switch, so that would simplify this file.